python/ssh-into-emr-cluster/requirements.txt

```diff
@@ -0,0 +1,2 @@
+aws-cdk-lib==2.49.0
```

Comment:
update to latest

Reply:
Would you be able to update this to latest. 2.49 is pretty old released back in Oct 27, 2022.

Reply:
^sure! this pattern was worked out a while back, so I guess that's why it's outdated

Let me upgrade CDK version, retest, and follow up here today